execution_engine_testing/tests/src/test/explorer/faucet.rs

```diff
@@ -814,7 +814,7 @@ fn faucet_costs() {
     // This test will fail if execution costs vary.  The expected costs should not be updated
     // without understanding why the cost has changed.  If the costs do change, it should be
     // reflected in the "Costs by Entry Point" section of the faucet crate's README.md.
```

Comment:
We need to update [the table mentioned in this comment](https://github.com/casper-network/casper-node/tree/dev/smart_contracts/contracts/explorer/faucet#costs-by-entry-point) too.